[OSDOCS-17003] IBM Z CQA work (#116100)
openshift/openshift-docs: modules/ibm-z-roce-network-cards.adoc

// Module included in the following assemblies:
//
// * installing/installing_ibm_z/ibmz-post-install.adoc

:_mod-docs-content-type: CONCEPT
[id="ibm-z-roce-network-cards_{context}"]
= RoCE network cards

[role="_abstract"]

> RoCE (RDMA over Converged Ethernet) network cards do not need to be enabled and their interfaces can be configured with the Kubernetes NMState Operator whenever they are available in the node. For example, RoCE network cards are available if they are attached in a z/VM environment or passed through in a {op-system-base} KVM environment.

Author: again to get below the 300 character limit: 
```suggestion
Configure RoCE (RDMA over Converged Ethernet) interfaces with the Kubernetes NMState Operator when RoCE network cards are available on a node. For example, the cards are available if they are attached in a z/VM environment or passed through in a {op-system-base} KVM environment.
```
